public/js/fxa-analytics.js

```diff
@@ -122,8 +122,17 @@ function getUTMNames() {
       },
     });
 
-    document.querySelectorAll(".send-ga-ping").forEach((el) => {
-      el.addEventListener("click", async(e) => {
+    // Temporarily send "View" and "Engage" pings for recommendation CTAs.
+    document.querySelectorAll(".recommendation-cta").forEach(cta => {
+      const eventLabel = cta.dataset.eventLabel;
+      ga("send", "event", "Breach Detail: Recommendation CTA", "View", eventLabel);
```

Comment:
This has the same "bug" as https://github.com/mozilla/blurts-server/pull/1429/files?file-filters%5B%5D=.js#r357364009 ... i.e., it fires the "View" event even if the recommendation CTA is behind "See additional recommendations".

Can we fix that for a merge + deploy on Monday?

Reply:
oooo, maybe a fancy `IntersectionObserver`?

Reply:
I will do the thing!

Reply:
Filed https://github.com/mozilla/blurts-server/issues/1445 for this.